<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>twitterpac_client2Class</class>
 <widget class="QWidget" name="twitterpac_client2Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1402</width>
    <height>867</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>twitterpac_client2</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1421</width>
     <height>861</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QGroupBox" name="groupBox">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>290</y>
       <width>271</width>
       <height>201</height>
      </rect>
     </property>
     <property name="title">
      <string>Sign in</string>
     </property>
     <widget class="QPushButton" name="pushButton_login">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_login {
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(29,155,240);
border-radius: 15px;
color:#ffffff;
}


</string>
      </property>
      <property name="text">
       <string>Log in</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_username">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border: none;
border-bottom: 3px solid rgb(29,155,240);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_password">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>161</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom: 3px solid rgb(29,155,240);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_register">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_register {
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(29,155,240);
border-radius: 15px;
color:#ffffff;
}




</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>130</y>
       <width>271</width>
       <height>361</height>
      </rect>
     </property>
     <property name="title">
      <string>Register</string>
     </property>
     <widget class="QPushButton" name="pushButton_submit_registration">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>252</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_submit_registration {
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(29,155,240);
border-radius: 15px;
color:#ffffff;
}


</string>
      </property>
      <property name="text">
       <string>Register</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_returnTo_login">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>312</y>
        <width>75</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton#pushButton_returnTo_login {
	font: 75 11pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(29,155,240);
border-radius: 15px;
color:#ffffff;
}


</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>290</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Aleady have an account?</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>70</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom: 3px solid rgb(29,155,240);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>120</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom: 3px solid rgb(29,155,240);</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>170</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom: 3px solid rgb(29,155,240);</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>72</x>
        <y>220</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border:none;
border-bottom: 3px solid rgb(29,155,240);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>100</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Password</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>150</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>First Name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>200</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;MS Shell Dlg 2&quot;;</string>
      </property>
      <property name="text">
       <string>Surname</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="twitterpac_client2.qrc"/>
 </resources>
 <connections/>
</ui>
